### ISSUE TITLE
PATH export on Windows is unquoted

### ISSUE THREAD

**Issue by filipesilva** [bazelbuild/rules_go#1619]
When trying to use `rules_go@0.13.0`, I receive a few errors that seem related to how the PATH export is templated.

```
$ bazel build ...
INFO: Build options have changed, discarding analysis cache.
INFO: Analysed 17 targets (33 packages loaded).
INFO: Found 17 targets...
ERROR: C:/users/kamik/_bazel_kamik/gq7cl633/external/io_bazel_rules_go/go/tools/builders/BUILD.bazel:33:1: error executing shell command: 'export GOROOT=$(pwd)/external/go_sdk && export GOROOT_FINAL=GOROOT && export PATH=C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/amd64 && external/go_sdk/bin/go.exe tool link -s -o baze...' failed (Exit 1)
/usr/bin/bash: -c: line 0: syntax error near unexpected token `('
/usr/bin/bash: -c: line 0: `export GOROOT=$(pwd)/external/go_sdk && export GOROOT_FINAL=GOROOT && export PATH=C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/amd64 && external/go_sdk/bin/go.exe tool link -s -o bazel-out/host/bin/external/io_bazel_rules_go/go/tools/builders/windows_amd64_stripped/compile.exe bazel-out/host/bin/external/io_bazel_rules_go/go/tools/builders/windows_amd64_stripped/compile~/go/tools/builders/compile.a'
INFO: Elapsed time: 52.795s, Critical Path: 0.17s
INFO: 0 processes.
FAILED: Build did NOT complete successfully
```

The key part here is that `export PATH=C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/amd64` should instead be `export PATH="C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/amd64"` (notice the quotes).

I also saw https://github.com/bazelbuild/rules_go/issues/1613 which lists the `--cpu=x64_windows --compiler=mingw-gcc --glibc=mingw` flags as mandatory, so I tried them:
```
$ bazel build ... --cpu=x64_windows --compiler=mingw-gcc --glibc=mingw
INFO: Build options have changed, discarding analysis cache.
INFO: Analysed 17 targets (0 packages loaded).
INFO: Found 17 targets...
ERROR: C:/users/kamik/_bazel_kamik/gq7cl633/external/io_bazel_rules_go/go/tools/builders/BUILD.bazel:33:1: error executing shell command: 'export GOROOT=$(pwd)/external/go_sdk && export GOROOT_FINAL=GOROOT && export PATH=C:/program files/git/mingw64/bin && external/go_sdk/bin/go.exe tool link -s -o bazel-out/host/bin/external/io_bazel...' failed (Exit 1)
/usr/bin/bash: line 0: export: `files/git/mingw64/bin': not a valid identifier
INFO: Elapsed time: 37.427s, Critical Path: 0.14s
INFO: 0 processes.
FAILED: Build did NOT complete successfully
```

Slightly different error but it looks like the same problem. `export PATH=C:/program files/git/mingw64/bin` should be `export PATH="C:/program files/git/mingw64/bin"`.

/cc @alexeagle

**Comment by filipesilva** [comment]
Added a fix in https://github.com/bazelbuild/rules_go/pull/1620. 

Are there other places I should be looking at?

I found this bit also:
https://github.com/bazelbuild/rules_go/blob/0a7b0ae47cb21e1aa77268b54b994eb5a5a574d5/go/tools/builders/protoc.go#L84

But I'm not too familiar with go; do you think the path string should also be quoted here?

**Comment by jayconrod** [comment]
Fixed by #1621
